### PR TITLE
packaging updates

### PR DESCRIPTION
**Pull request by marjo-luc**
To test install:

pip install maap-help-jupyter-extension==1.2.0

### CODE REVIEW

File: package.json
Comment (grallewellyn) [inline package.json:93]: Did removing typescript from the package.json help solve problems? Could it potentially hurt us later down the road not pinning the typescript version since we are getting so many typescript errors trying to run the readme commands?
I tried this for the jupyter-server-extension library and got `/bin/sh: tsc: command not found`

Comment (marjo-luc) [reply package.json:93]: It should install the version specified by jupyterlab-tour, which is: `"typescript": "~4.1.3"`. I don't think we want to pin more than the major and minor version numbers. But if you're running into issues building this, we might want to explicitly install that version of typescript.

Comment (grallewellyn) [reply package.json:93]: Got this working by adding "typescript": "~4.1.3" (if I tried without this I got `/bin/sh: tsc: command not found` for maap help), using the environment.yml that you gave me, and using the node version you used: v16.10.0
I will approve if adding that works for you too 
Also, what are your thoughts on adding a .nvmrc file with the node version 16.10.0?

Comment (grallewellyn) [reply package.json:93]: Note that I was able to replicate this a second time following the same method I described above with a new conda environment (but still created from your environment.yml)

Comment (marjo-luc) [reply package.json:93]: I've added the explicit install of typescript and everything looks good on my end. I will publish a new version of the extension after merging.

File: maap_help_jupyter_extension/_version.py
Comment (grallewellyn) [inline maap_help_jupyter_extension/_version.py:1]: Do we want to be committing this file even though it is in the .gitignore?

Comment (marjo-luc) [reply maap_help_jupyter_extension/_version.py:1]: This file is autogenerated and was committed to the repo before it was added to .gitignore. I can delete it. We should probably go through and clean up the repo to remove the other binaries and configs for services we aren't using in another ticket.

**Review by grallewellyn — APPROVED** [review]
Looks good now, thanks for making those changes!

**Pull request merged**
